src/data/points/points.ts

```diff
@@ -67,7 +67,7 @@ export async function getStepPoints(stepId: number): Promise<PointOutput[]> {
 export async function updatePoint(stepId: number, pointId: number, data: Partial<PointInput>): Promise<PointOutput> {
 
 	try {
-		const response = (await request(`/trips/${stepId}/points${pointId}`, "PUT", data));
+		const response = (await request(`/steps/${stepId}/points${pointId}`, "PUT", data));
```

Comment:
Not the right endpoint
Delete the parameter "stepId" et change endpoint to "/points/:pointId"

https://github.com/La-Sectoblique/septotrip-api/blob/36cc241fc79522a90f378c89d959403b7eec1171/src/routes/protected/points.ts#L15

Reply:
done